cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureEvaluationTest.java

```diff
@@ -158,7 +165,8 @@ void proportionMeasureEvaluation() throws Exception {
 
         Measure measure = proportion_measure();
```

Comment:
Question: do we have a test that runs this scenario:
- reportType=subject
- subject=null (or value that returns more than one patient)
and creates a MeasureReport?

Reply:
New Measure test added for a Measure with multiple SDEs.